### ISSUE TITLE
Notification not working

### ISSUE THREAD

**Issue by kmvignesh** [Almoullim/background_location#114]
I have updated the project to null-safty. that time i updated the version to
```
background_location: ^0.6.1
permission_handler: ^8.1.4+2
```
Then i started testing the app and found background location tracking notification not showing. Then I downgraded the plugin version to
```
background_location: ^0.5.0
permission_handler: ^6.1.3
```
Now its working. So please check and fix the issue.

**Comment by ambernardino** [comment]
On 0.6.1, did you set the android notification?

BackgroundLocation.setAndroidNotification(
	title: "Notification title",
        message: "Notification message",
        icon: "@mipmap/ic_launcher",
);

**Comment by kmvignesh** [comment]
yes I did. but I am using await keyword before that. I saw some where asked to change the icon. So I tried using other icon also. Still didn't work. Same code is working with 0.5.0

**Comment by jesussmile** [comment]
same notification not working for me too..

**Comment by MoralCode** [comment]
Is this still an issue?

**Comment by MoralCode** [comment]
may be related to #147

**Comment by Almoullim** [comment]
fixed in https://github.com/Almoullim/background_location/commit/52ef3db3e572596bb0e879dc57aa201fe160143e version 0.12.0

**Comment by MoralCode** [comment]
@Almoullim did you confirm that this was actually related to notification importance? it definitely seems plausible but idk if theres enough info in the original PR to know for sure

**Comment by Almoullim** [comment]
@MoralCode  if not fixed in 0.12 then its a duplicate of https://github.com/Almoullim/background_location/issues/131, no point in keeping in open.
